### ISSUE TITLE
Full list of available boundaries listed in docs is incomplete

### ISSUE THREAD

**Issue by andy-esch** [CartoDB/bigmetadata#382]
The list here is incomplete: https://carto.com/docs/carto-engine/data/glossary/#boundary-ids

For example, Mexico has Municipios (`mx.inegi.municipio`) and Localidades (`mx.inegi.localidad_urbana_y_rural_amanzanada`).

**Comment by javitonino** [comment]
I think we should remove this page and just link to the catalog. It is very hard to keep this kind of duplicated information up to date. In fact, I think this is something planned for the documentation revamp project.

**Comment by csobier** [comment]
We are pulling this doc directly from the observatory-extension repo: https://github.com/CartoDB/observatory-extension/blob/develop/doc/boundary_functions.md

**Comment by andy-esch** [comment]
Yeah, I agree it's best to remove it since it's a duplicate that's maintained separately. One thing I like about this page, though, is that it lists all the boundary IDs together.. it makes discovery easier than going down a few levels for each country/region in the bigmetadata docs page.

**Comment by csobier** [comment]
Tagging @inigomedina for transparency. (We already have an issue about this as part of the [docs redesign](https://github.com/CartoDB/docs/projects/7) project). Let me know if there is something we should do before then?

**Comment by inigomedina** [comment]
Yes, the new approach takes the direction @javitonino suggests: just let each CARTO ecosystem part do its own job, in a separated manner, and users would find the best way of combining these. That means avoiding duplicated content and creating just links between each part.

Since the plan is to release new developer documentation at the end of January, we shouldn't make any change now in these contents.
